File: App_Flutter/lib/data/models/BreakingNewsModel.dart
// ignore_for_file: file_names

import 'package:news/utils/strings.dart';

class BreakingNewsModel {
  String? id, image, title, desc, contentType, contentValue, lanId, totalViews;

  BreakingNewsModel({this.id, this.image, this.title, this.desc, this.contentValue, this.contentType, this.lanId, this.totalViews});

  factory BreakingNewsModel.fromJson(Map<String, dynamic> json) {
    return BreakingNewsModel(
        id: json[ID],
        image: json[IMAGE],
        title: json[TITLE],
        desc: json[DESCRIPTION],
        contentValue: json[CONTENT_VALUE],
        contentType: json[CONTENT_TYPE],
        lanId: json[LANGUAGE_ID],
        totalViews: json[TOTAL_VIEWS]);
  }
}

File: Admin_Panel/app/Models/BreakingNews_Model.php
<?php

namespace App\Models;

use CodeIgniter\Model;

class BreakingNews_Model extends Model {

    protected $table = 'tbl_breaking_news';
    protected $allowedFields = ['title', 'image', 'content_type', 'content_value', 'description','language_id'];

}
